### ISSUE TITLE
I cant enable the addon 

### ISSUE THREAD

**Issue by camilod764** [mika-f/blender-drag-and-drop#55]
### What happened?

After download the addon and install it, I cant find it to enable it and use it.

Someone can help me how to enable, I search in several ways but it doesnt look.
Also I close the program and re install several times but it doesnt work
I use 3.6.1 and also a old version

![image](https://github.com/mika-f/blender-drag-and-drop/assets/143615230/5d839d59-a915-464a-8dd8-934daba2028e)

### Add-on Version

2.8.0

### Blender versions

3.6.1

**Comment by MischaStruggles** [comment]
Hi, so I had to re-install the 3.5 version of Blender. This Add-on no longer works on 3.6 and onwards.

**Comment by kloworks** [comment]
same I am using blender 3.6.2 and when I drag nothing happen. 
using "DragAndDropSupport-v2.8.0"

+ thanks for creating this addon

**Comment by mika-f** [comment]
This add-on does not support Blender 3.6.2 yet.
However, it's strange that it doesn't work with Blender 3.6.0 and Blender 3.6.1.
Could you please reply below with reference to which variant of Blender you are using?

Example: 

* [ ] Blender 3.6.0 - Windows Installer
* [ ] Blender 3.6.0 - MSIX
* [x] Blender 3.6.0 - ZIP
* [ ] Blender 3.6.0 - Steam
* [ ] Blender for Artists 3.6.0 - Windows Installer
* [ ] Blender for Artists 3.6.0 - ZIP

**Comment by jayliu50** [comment]
I was also seeing this exact issue. But then I realized that after downloading the .zip, UNZIP and there will be ANOTHER .zip file inside which then you can use to install. @camilod764 @kloworks

**Comment by akshu1111** [comment]
Yes for all blender plugins the __init__.py has to be directly inside a folder, as in: ./addons/your_plugin_folder/__init__.py
Now if you zip a folder that has __init__.py inside it, blender will extract it that way only. If theres an extra folder OR if you directly selected all the source and zipped it (say from a context menu in the file manager/explorer), in that case blender will still extract it but there wont be any folder to find. Same goes for zipping a folder that has the source folder inside it
